src/pykeen/pipeline.py

```diff
@@ -727,12 +741,13 @@ def pipeline(  # noqa: C901
         optimizer_kwargs = {}
 
     # Log optimizer parameters
-    result_tracker.log_params({'class': optimizer, 'kwargs': optimizer_kwargs}, prefix='optimizer')
+    result_tracker.log_params({'class': optimizer.__name__, 'kwargs': optimizer_kwargs}, prefix='optimizer')
     optimizer_instance = optimizer(
         params=model_instance.get_grad_params(),
         **optimizer_kwargs,
     )
 
+    result_tracker.log_params(params=dict(cls=training_loop.__name__), prefix='training_loop')
```

Comment:
inconsistent 'cls' versus 'class'

Reply:
Resolved in 9f73f27 

I used `cls` since it does not collide with Python builtins and hence can be used with the `dict` way of defining a dictionary.

I chose `dict` for initialization of the dictionary since it ensures that all keys are strings.

Reply:
Okay sounds good!